lib/dash/dash_parser.js

```diff
@@ -1489,7 +1505,8 @@ shaka.dash.DashParser = class {
     const adaptationSetNodes =
         TXml.findChildren(periodInfo.node, 'AdaptationSet');
     const adaptationSets = adaptationSetNodes
-        .map((node) => this.parseAdaptationSet_(context, node))
+        .map((node, position) =>
+          this.parseAdaptationSet_(context, position, node))
```

Comment:
```suggestion
            this.parseAdaptationSet_(context, position, node))
```

Reply:
well lint check is not accepting it
  1510:1  error  Expected indentation of 10 spaces but found 12  indent

Reply:
ok!

Reply:
@avelad ... are you sure ...